be/src/cloud/cloud_tablet.cpp

```diff
@@ -538,12 +553,16 @@ void CloudTablet::remove_unused_rowsets() {
         auto& key_ranges = std::get<1>(*it);
         tablet_meta()->delete_bitmap().remove(key_ranges);
         it = _unused_delete_bitmap.erase(it);
+        removed_delete_bitmap_num++;
     }
 
-    if (!_unused_rowsets.empty() || !_unused_delete_bitmap.empty()) {
+    if (removed_rowsets_num > 0 || removed_delete_bitmap_num > 0) {
```

Comment:
why not just LOG without this check?
what's the cost?

Reply:
> why not just LOG without this check? what's the cost?

done